t/140-ssl-c-api.t

```diff
@@ -11,6 +11,8 @@ my $openssl_version = eval { `$NginxBinary -V 2>&1` };
 if ($openssl_version =~ m/built with OpenSSL (0|1\.0\.(?:0|1[^\d]|2[a-d]).*)/) {
     plan(skip_all => "too old OpenSSL, need 1.0.2e, was $1");
 
+} elsif ($openssl_version =~ m/AWS-LC/) {
```

Comment:
Hmm, how did you test this? Curious. Also interested in this AWS-LC lib for SSL. Can help with stuff if you want.

Reply:
> Hmm, how did you test this? Curious. Also interested in this AWS-LC lib for SSL. Can help with stuff if you want.

Using CI, you can check https://app.travis-ci.com/github/openresty/lua-nginx-module/jobs/625874178 and https://app.travis-ci.com/github/openresty/lua-nginx-module/jobs/625875719